cmd/cosign/cli/sign/sign.go

```diff
@@ -205,9 +205,11 @@ func signDigest(ctx context.Context, digest name.Digest, payload []byte, ko KeyO
 		}
 	}
 
+	// TODO(dekkagaijin): hoist the creation of these into SignerFromKeyOpts
 	var s icos.Signer
-	s = ipayload.NewSigner(sv, nil, nil)
+	s = ipayload.NewSigner(sv, nil, nil, nil, nil)
```

Comment:
This has a code smell to it.  We're going something wrong here 😅

Reply:
Potential solutions: 1) populate the struct directly 2) `option`s 3) hoist the cert stuff into a higher-level wrapper